apps/client/src/components/FormInput.tsx

```diff
@@ -0,0 +1,48 @@
+import TextField from '@mui/material/TextField';
+import React, { useState } from 'react';
+
+interface FormInputProps {
+  label: string;
+  type: string;
+  name: string;
+  value: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  icon?: string;
+  placeholder?: string;
+  required?: boolean;
+  helperText?: string;
+  error?: boolean;
+}
+
+const FormInput: React.FC<FormInputProps> = ({
+  label,
+  type,
+  name,
+  value,
+  onChange,
+  placeholder,
+  required = false,
+  helperText,
+  error = false,
+}) => {
+  return (
+    <div className="">
+      <div>
```

Comment:
What are these divs doing?
In React, every div is a different DOM node, so this makes the final DOM tree dirty without reason